src/actions/config.ts

```diff
@@ -0,0 +1,18 @@
+import {VoyagerConfig} from '../models/config';
+import {ReduxAction} from './redux-action';
+
+export type ConfigAction = SetConfig;
+
+export const SET_CONFIG = 'SET_CONFIG';
+export type SetConfig = ReduxAction<typeof SET_CONFIG, {
+  config: VoyagerConfig
+}>;
+
+export function setConfig(config: VoyagerConfig): SetConfig {
```

Comment:
> do you mean you would just dispatch the object (result of the action creator) directly from the component? 

Yes

> Any examples with a payload?

Most other shelf action follow this pattern too.  

>  I guess i like the encapsulation of the action creator in case the action actually ends up being more 'steps' internally than a component is concerned with.

I think this is a valid point.  However, given the rest of the app use the object directly for action with one step, it might be better to be consistent.